package.json

```diff
@@ -30,13 +30,15 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-polyfill": "^6.26.0",
     "c3": "^0.4.18",
+    "chart.js": "^2.7.2",
     "eslint-plugin-prettier": "^2.6.0",
     "html-react-parser": "^0.4.1",
     "lodash": "^4.17.4",
     "prop-types": "^15.6.0",
     "react": "^16.1.1",
     "react-ace": "^5.10.0",
     "react-bootstrap": "^0.31.5",
+    "react-chartjs-2": "^2.7.4",
```

Comment:
I suppose react-chartjs-2 is using chart.js as an internal dependency. Why have you added it?